lib/foodcritic/rules/fc078.rb

```diff
@@ -0,0 +1,135 @@
+rule "FC078", "Ensure cookbook shared under an OSI-approved open source license" do
+  tags %w{supermarket}
```

Comment:
I think we're going to want another tag here so people can disable open source type checks in their internal CI systems. Perhaps 'community' or 'opensource'